app/screens/channel_info/channel_info_header.js

```diff
@@ -172,6 +177,29 @@ export default class ChannelInfoHeader extends React.PureComponent {
                         {displayName}
                     </Text>
                 </View>
+                {isCustomStatusEnabled && type === General.DM_CHANNEL && customStatus.emoji &&
+                    <View
+                        style={[style.row, style.customStatusContainer]}
+                        testID={`${testID}.custom_status`}
+                    >
+                        <Text
+                            style={style.iconContainer}
+                            testID={`custom_status.emoji.${customStatus.emoji}`}
+                        >
+                            <Emoji
+                                emojiName={customStatus.emoji}
+                                size={20}
+                            />
```

Comment:
Why not CustomStatusEmoji?

Reply:
Using CustomStatusEmoji component will require us to pass user id and then finding the emojiName which seems like a waste as we already have the value of emojiName here.